.github/workflows/run-unit-tests.yml

```diff
@@ -28,6 +28,7 @@ jobs:
         uses: ./.github/actions/build-khiops
         with:
           preset-name: ${{ matrix.build-setup.cmake-preset }}
+          targets: "--target norm_test --target Parallel_test --target learning_test"
```

Comment:
We can just write `--target norm_test Parallel_test  learning_test`
But it is ok like this

Reply:
I changed the "API" of the action : now it receives a list of targets ... (no `--target` necessary)